Modules/_zstd/zstddict.c

```diff
@@ -120,9 +120,9 @@ ZstdDict_dealloc(PyObject *ob)
 
 PyDoc_STRVAR(ZstdDict_dictid_doc,
 "the Zstandard dictionary, an int between 0 and 2**32.\n\n"
```

Comment:
The following sentences are properly capitalised:
```suggestion
"The Zstandard dictionary, an int between 0 and 2**32.\n\n"
```